README.md

```diff
@@ -7,9 +7,12 @@ In order to test changes to the game, you need to be able to copy the code out o
 #### Prerequisites
 - Create a BGA developer account on https://studio.boardgamearena.com/.
 - Create your own BGA Studio project for Innovation at https://studio.boardgamearena.com/studio. Make sure to suffix the new project name with your username (e.g. `InnovationJohnSmith`).
+- Optionally Install github desktop (https://desktop.github.com/) -> this is a simplier way of using git if you want to skip the commandline
 - Clone this repository onto your computer (`git clone https://github.com/micahstairs/bga-innovation.git innovation`). You'll make your life easier if the directory is named `innovation` instead of `bga-innovation`.
 - Clone https://github.com/elaskavaia/bga-sharedcode onto your computer (`git clone https://github.com/elaskavaia/bga-sharedcode.git`).
 - Install PHP (https://windows.php.net/download#php-7.3).
+- Install SASS (https://sass-lang.com/install)
```

Comment:
Maybe add a note that this is only needed if they are going to make changes to SCSS.